src/averageAges.js

```diff
@@ -36,8 +41,22 @@ function calculateMenAverageAge(people, century) {
  *
  * @return {number}
  */
+
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  let women = people;
+
+  if (withChildren) {
+    women = people.filter(person =>
```

Comment:
краще назвати інтуїтивніше
```suggestion
    womenAges = people.filter(person =>
```